### PR TITLE
Add minor changes

### PR DESCRIPTION
**Pull request by jtapia**

### CODE REVIEW

**Review by ozmar-11 — APPROVED** [review]
LGTM

**Comment by jtapia** [comment]
@softr8 @jaimelr

**Review by jaimelr — APPROVED** [review]

**Comment by jtapia** [comment]
@jaimelr @ozmar-11 could you confirm if this update is still working on your actual implementations, please?

**Comment by ozmar-11** [comment]
Sorry @jtapia I used to use this gem on TireGet MX but as long as I leave the project I haven't touch it again, but I'll test it and let you know, is there some specific detail that I should pay attention?

**Comment by jtapia** [comment]
@ozmar-11 not really, only to check that the payment process is working properly and the gem references are ok, thanks

File: app/models/spree/conekta_oxxo_payment.rb [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
+    def authorize(_money, source, _gateway_options)
       ::Conekta.api_key = api_key
       ::Conekta.api_version = "2.0.0"
 
-      order = Spree::Order.find_by(number: creditcard.number)
+      order = Spree::Order.find_by(number: source.number)
 
       begin
         response = ::Conekta::Order.create(payload(order))
         creditcard.update_attribute :conekta_order_id, response.id
```
Comment (edman193) [inline app/models/spree/conekta_oxxo_payment.rb]: @jtapia I think there is an undefined variable here, `creditcard` was replaced by `source`.

Comment (jtapia) [reply app/models/spree/conekta_oxxo_payment.rb]: thanks @edman193, fixed here https://github.com/magma-labs/solidus_oxxo_pay/pull/9/commits/050aa4e20cc025e5e23ff85e66146be6cf686f59#diff-10ab729171451fa861198bab233a83ceR15

**Review by edman193 — APPROVED** [review]
LGTM

**Pull request merged**